### PR TITLE
Release v1.9 (#1216)

### PR DESCRIPTION
**Pull request by marecar3**
Brings the v1.9.0 release to develop.

Update release notes:

- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -126,6 +126,12 @@ jobs:
         paths:
         - react-native-aztec/ios/Carthage
         - ~/.rncache
+    - run:
```
Comment (marecar3) [inline .circleci/config.yml]: Hey, @JavonDavis @etoledom I have resolved merge conflicts and I need to help from someone of you to check if everything is on the place in this file :) Thanks! :)

File: __device-tests__/pages/editor-page.js [hunk 2]
```diff
@@ -72,6 +72,13 @@ export default class EditorPage {
 		return await this.driver.elementByXPath( blockLocator );
 	}
 
+	async getTitleElement() {
```
Comment (marecar3) [inline __device-tests__/pages/editor-page.js]: Hey, @JavonDavis @etoledom I have resolved merge conflicts and I need to help from someone of you to check if everything is on the place in this file :) Thanks! :)

File: __mocks__/styleMock.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
+/** @flow
```
Comment (marecar3) [inline __mocks__/styleMock.js]: Hey, @JavonDavis @etoledom I have resolved merge conflicts and I need to help from someone of you to check if everything is on the place in this file :) Thanks! :)

Comment (etoledom) [reply __mocks__/styleMock.js]: This file should actually have been removed: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1215/files

File: __device-tests__/pages/editor-page.js (hunk 2 above)
Comment (etoledom) [reply __device-tests__/pages/editor-page.js]: `getTitleElement` seems to be duplicated. That's probably why the UI Tests are failing.
@JavonDavis might have a better context to point out which is the correct one. (Probably the one already on dev?)

File: .circleci/config.yml (hunk 1 above)
Comment (JavonDavis) [reply .circleci/config.yml]: Hey @marecar3 seems you duplicated the `Build (if needed)` step here, and the bundle step here, removing your addition should resolve that

File: __device-tests__/pages/editor-page.js (hunk 2 above)
Comment (JavonDavis) [reply __device-tests__/pages/editor-page.js]: @etoledom you're right here, the one already in dev is the one we should keep, this duplicate function can be removed

File: __device-tests__/helpers/test-data.js [hunk 4]
```diff
@@ -87,3 +87,21 @@ exports.imageCompletehtml = `<!-- wp:image {"id":1} -->
 <!-- wp:paragraph -->
 <p>The finer continuum interprets the polynomial rabbit. When can the geology runs? An astronomer runs. Should a communist consent?</p>
 <!-- /wp:paragraph -->`;
+
+exports.imageCaption = `C'est la vie my friends`;
```
Comment (JavonDavis) [inline __device-tests__/helpers/test-data.js]: @marecar3 these attributes are already exported above this, you can remove this addition and just use the above.

File: __mocks__/styleMock.js (hunk 3 above)
Comment (marecar3) [reply __mocks__/styleMock.js]: Thanks @etoledom, removed!

File: __device-tests__/pages/editor-page.js (hunk 2 above)
Comment (marecar3) [reply __device-tests__/pages/editor-page.js]: Thanks @etoledom @JavonDavis , removed!

File: __device-tests__/helpers/test-data.js (hunk 4 above)
Comment (marecar3) [reply __device-tests__/helpers/test-data.js]: Thanks @JavonDavis, removed!

File: .circleci/config.yml (hunk 1 above)
Comment (marecar3) [reply .circleci/config.yml]: Thanks @JavonDavis , removed!

**Comment by marecar3** [comment]
Hey @etoledom, @JavonDavis I resolved all comments from this PR and build is kicked again.
With a bit of luck, we will see green light again :) 
Thanks a lot for the comments!

**Pull request merged**